be/src/vec/functions/function_string.h

```diff
@@ -2740,6 +2741,51 @@ class FunctionStringParseUrl : public IFunction {
     }
 };
 
+class FunctionUrlDecode : public IFunction {
```

Comment:
the function should be in url/function_url.cpp

Reply:
> the function should be in url/function_url.cpp

I will migrate it in the next PR.